BuildResidentialHPXML/measure.rb

```diff
@@ -4224,6 +4224,7 @@ def self.set_building_construction(hpxml_bldg, args)
     hpxml_bldg.building_construction.conditioned_building_volume = conditioned_building_volume
     hpxml_bldg.building_construction.average_ceiling_height = args[:geometry_average_ceiling_height]
     hpxml_bldg.building_construction.residential_facility_type = args[:geometry_unit_type]
+    hpxml_bldg.building_construction.number_of_units_in_building = args[:geometry_building_num_units]
```

Comment:
Ah, I see the issue. This is an optional input, so you have to check if it was provided or not first. I think this should work?

```suggestion
    if args[:geometry_building_num_units].is_initialized
      hpxml_bldg.building_construction.number_of_units_in_building = args[:geometry_building_num_units].get
    end
```